torch/csrc/jit/python/init.cpp

```diff
@@ -1870,6 +1870,13 @@ void initJITBindings(PyObject* module) {
             ss << self;
             return ss.str();
           })
+      .def(
+          "__repr__",
+          [](FunctionSchema& self) {
+            std::stringstream ss;
```

Comment:
Nit: std::stringstream is pretty inefficient at string construction, probably fine, but can we use a faster method from libfmt?